### PR TITLE
Pd routing

### PR DESCRIPTION
**Pull request by ThinkCreatively**
~ Added Carousel Via a npm package
~ Carousel fits images accordingly and renders the images from the API
~ Added more css to the material ui file for styling
~ (WORK IN PROGRESS) Added a function to selectors that gets data from api and adds to a array for ease of access (Susceptible to change or deletion when I get back to Selectors)
Screen Shot 2021-06-11 at 12 20 32 PM

### CODE REVIEW

File: client/src/components/product-display/CategoryPriceName.jsx [hunk 1]
```diff
@@ -23,6 +24,7 @@ const CategoryPriceName = (props) => {
             {product.name}
           </Typography>
           <Typography variant="body2" color="textSecondary" component="p" align="left">
+            $
             {product.default_price}
```
Comment (Pix3lAssassin) [inline client/src/components/product-display/CategoryPriceName.jsx:28]: might consider templating this {`$${product.default_price}`}

File: client/src/components/product-display/MaterialUi.jsx [hunk 2]
```diff
@@ -45,43 +46,51 @@ const useStyles = makeStyles((theme) => ({
...
+  //   color: 'white',
+  //   backgroundColor: 'red',
+  //   border: '2px solid black'
+  // },
+  // grid5: {
+  //   textAlign: 'center',
+  //   color: 'white',
+  //   backgroundColor: 'gray',
+  //   border: '2px solid black'
+  // }
```
Comment (Pix3lAssassin) [inline client/src/components/product-display/MaterialUi.jsx:93]: consider removing as you have a copy with grid

File: client/src/components/product-display/StyleThumbs.jsx [hunk 3]
```diff
@@ -6,42 +6,46 @@ import useStyles from './MaterialUi.jsx';
 
 const StyleThumbs = (props) => {
```
Comment (Pix3lAssassin) [inline client/src/components/product-display/StyleThumbs.jsx:7]: consider destructuring props

**Review by Pix3lAssassin — APPROVED** [review]

File: client/src/components/product-display/ImageGallery.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      <img
+        src={props.src}
+        alt="Product Style"
+        className={classes.img}
+      />
+    </Paper>
+  );
+};
+
+export default ImageGallery;
```
Comment (mhhutton) [inline client/src/components/product-display/ImageGallery.jsx:38]: Nice use of carousel layout

**Review by mhhutton — APPROVED** [review]

**Pull request merged**